Prime_Number/Prime_Number.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdbool.h>
+ 
+bool Prime (int num, int i) 
+{ 
+    // Base cases 
+    if (i <= 2) 
+        return true; 
+    // Checking if i is a divisor or not
+    if (num % i == 0) 
+        return false; 
+      
+    // Calling recursive function
+    return Prime(num, i - 1); 
+} 
+  
+// Driver Program 
+int main() 
+{ 
+    int num;    
+    scanf("%d", &num); 
+
+    if (Prime(num, num - 1)) 
```

Comment:
A corner case, what if I input num as 1? The function will return true. But it should be false. Isn't it?